### PR TITLE
Removed bootstrapped project. Closes #2

### LINKED ISSUE #2: Clean out bootstrapped project

### PR DESCRIPTION
**Pull request by patstockwell**
- Did some stuff
:pear: @zosia-fabijanska

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
```
Comment (zosia-fabijanska) [inline src/index.js:3]: Why did you remove this file?

Comment (patstockwell) [reply src/index.js:3]: Because we are cleaning out first 🌮

Comment (zosia-fabijanska) [reply src/index.js:3]: That's a good idea! Well done! 🥇 🦑

**Review by zosia-fabijanska — APPROVED** [review]
LGTM!
